modules/administration-guide/attachments/migration/rollback.sh

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+K8S_CLI=${K8S_CLI:-oc}                                                           # {orch-cli}
```

Comment:
Given this is Che/upstream, shouldn't the default value be `kubectl`?

Reply:
No, it is OpenShift related doc. So `oc` for both cases.

Reply:
OK, in that case, we shouldn't reference `{orch-cli}` because in upstream, it resolves to `kubectl`.